docs/source/playbooks.rst

```diff
@@ -142,20 +154,31 @@ path; for example, ``/usr/lpp/IBM/zoautil/lib/``.
 The value for the property **LIBPATH** is both the path to the Python libraries
 on the target and the ZOA Utilities Python library path separated by
 colons ``:``; for example,
-``/usr/lpp/IBM/zoautil/lib/:/usr/lpp/rsusr/python36/lib:/lib:/usr/lib:.``.
+``/usr/lpp/IBM/zoautil/lib/:/usr/lpp/IBM/cyp/v3r8/pyz/lib:/lib:/usr/lib:.``.
 
 The value for the property **PATH** is the ZOA utilities BIN path and the Python
 interpreter path; for example,
-``/usr/lpp/IBM/zoautil/bin:/usr/lpp/rsusr/python36/bin/python:/bin``.
+``/usr/lpp/IBM/zoautil/bin:/usr/lpp/IBM/cyp/v3r8/pyz/bin/python3:/bin``.
+
+The value for the property **_CEE_RUNOPTS** is the invocation Language
+Environment® runtime options for programs and used by Python, for example,
+``'FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)'``.
+
+The value for properties **__TAG_REDIR_ERR**, **_TAG_REDIR_IN**,
```

Comment:
I don't think this is accurate, we actually don't even support using bash on managed node, and the variables are meaningless on control node.